src/components/forms/file-dropper/file-dropper.tsx

```diff
@@ -45,11 +47,24 @@ interface FileDropperProps {
   /**
    * (Optional) Categories that the user can add to each uploaded file (e.g. 'CV' or 'Cover Letter')
    * */
-  fileCategories?: string[];
+  fileCategories?: { value: string; label: string }[];
   /**
    * (Optional) Pass in a valid illustration keyword to show the according illustration.
    */
   illustration?: IllustrationType;
+  /**
+   * (Optional) label, which is displayed above
+   */
+  label?: string;
+  /**
```

Comment:
Adding some whitespace above makes it a bit more clear where the comment belongs to:

```ts
  /**
   * (Optional) Pass in a valid illustration keyword to show the according illustration.
   */
  illustration?: IllustrationType;

  /**
   * (Optional) label, which is displayed above
   */
  label?: string;

  /**
   * Use Controller Props, to get the form to work
   */
  setValue: UseFormSetValue<FormDataProps>;
```

Reply:
👍